docs/pages/auto-discovery/databases.mdx

```diff
@@ -1,183 +1,219 @@
 ---
-title: AWS Database Auto-Discovery
-description: How to configure Teleport to discover AWS-hosted databases.
+title: Database Discovery
+description: Detailed guides for configuring database discovery.
 ---
 
-Database discovery involves two components.
+Teleport can be configured to discover databases automatically and register
+them with your Teleport cluster.
 
-The first, the Discovery Service, is responsible for watching your cloud
-provider and checking if there are any new databases or if there have been any
-modifications to previously discovered databases.
+## Supported clouds
 
-The second, the Database Service, monitors the databases created by the
-Discovery Service and proxies communications between users and the database
-servers hosted by the cloud provider.
+- [AWS](./databases/aws.mdx): Discovery for AWS databases.
+{/* TODO(gavin): Add an Azure discovery guide and permission reference */}
 
-In this guide, you will configure a Teleport Discovery Service instance to
-discover AWS-hosted databases automatically.
+## How database discovery works
 
-## Prerequisites
+Database auto-discovery consists of the following components:
 
-(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+1. One or more Teleport Discovery Services polling APIs to discover and register
+  databases with your Teleport cluster.
+1. One or more Teleport Database Services forwarding database connections to
+  the discovered databases.
 
-- An AWS account with permissions to create and attach IAM policies.
-- One or more database servers hosted by AWS database services.
-- A host to run the Teleport Discovery Service.
+Both Discovery and Database service can be configured in the same Teleport
+process or separate processes.
+We recommend running them as separate processes to decouple database discovery
+from database access.
 
-## Step 1/4. Generate a join token
+### Discovery Service
```

Comment:
I would use a more explicit organization for this section onward, adding more descriptive headings.

It wasn't clear to me, for example, that this section is about configuration as well as architecture.

I would add a new heading every 200 words or so.

Reply:
update: https://github.com/gravitational/teleport/compare/9d2ef33f87d5bf180c306df2fa79107388dca996..c015df7542ccb87079e985afb65f4fe6bb2f825c

What do you think?